### ISSUE TITLE
App_Offline - Linux/Apache

### ISSUE THREAD

**Issue by mbuchok** [dotnet/core#2039]
# Issue Title

app_offline equivalent for Linux/Apache

# General

Perhaps I'm still a little new to Linux, but is there an app_offline equivalent for dotnet core hosted in a Linux environment... specifically using CentOS and Apache?

**Comment by karelz** [comment]
What is app_offline? Is it ASP.NET concept? Like this one: https://stackoverflow.com/questions/1153449/asp-net-2-0-how-to-use-app-offline-htm

**Comment by mbuchok** [comment]
Yes, just like that concept ... but doesn't work in a Linux environment since it appears to relate strictly to IIS.
